ngraph/frontend/onnx_import/src/core/node.cpp

```diff
@@ -238,6 +238,12 @@ namespace ngraph
             return m_pimpl->template get_attribute_value<std::int64_t>(name, default_value);
         }
 
+        template <>
+        int Node::get_attribute_value(const std::string& name, int default_value) const
+        {
+            return m_pimpl->template get_attribute_value<int>(name, default_value);
+        }
+
```

Comment:
Do we really need this helper?
Until now we have always used int64_t specialization when we tried to get an int attribute.